pysatModels/models/ucar_tiegcm.py

```diff
@@ -159,7 +159,13 @@ def load(fnames, tag=None, inst_id=None, **kwargs):
 
     """
 
-    data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False)
+    # TODO(#114): eventually remove support for multiple pysat versions
+    if hasattr(pysat.utils, 'load_netcdf4'):
```

Comment:
The current pysat release and `load_meta_branch` both have `pysat.utils.load_netcdf4`. A check on `io.load_netcdf` would work.

Reply:
Oops. I started a review on a single commit.... Will check over the full thing.